### PR TITLE
Release 1.0.0-rc.1

### PR DESCRIPTION
**Pull request by dmtrKovalenko**

### CODE REVIEW

File: lib/package.json [hunk 1]
```diff
@@ -34,17 +34,22 @@
   "peerDependencies": {
     "classnames": "^2.2.5",
     "material-ui": "^1.0.0-beta.33",
-    "moment": "^2.19.2",
     "prop-types": "^15.6.0",
     "react": "^16.2.0",
     "react-dom": "^16.2.0"
   },
+  "externalDependencies": {
```
Comment (cherniavskii) [inline lib/package.json]: didn't find any info about this `externalDependencies` key. Shouldn't `moment` and `date-fns` be in `peerDependencies`? I'm not sure if it's possible to mention `one of` there though...

File: lib/package.json [hunk 2]
```diff
@@ -79,28 +84,29 @@
...
-    "eslint-plugin-import": "^2.7.0",
+    "eslint-plugin-import": "^2.9.0",
     "eslint-plugin-jsx-a11y": "^6.0.2",
-    "eslint-plugin-react": "^7.5.1",
+    "eslint-plugin-react": "^7.7.0",
     "fs-extra": "^4.0.2",
     "glob": "^7.1.2",
     "jest": "^21.2.1",
-    "material-ui": "^1.0.0-beta.33",
+    "material-ui": "^1.0.0-beta.35",
```
Comment (cherniavskii) [inline lib/package.json]: Are all breaking changes of mui 34 already resolved? If so, mui needs to be updated in `peerDependencies` too

Comment (cherniavskii) [reply lib/package.json]: also, there are no breaking changes in mui 35, so this release will support mui 34 as well

File: lib/package.json (hunk 1 above)
Comment (dmtrKovalenko) [reply lib/package.json]: Its not possible, and good that you mention - I forgot to add error inside the utils file, that some of required library not installed

File: lib/package.json (hunk 2 above)
Comment (dmtrKovalenko) [reply lib/package.json]: We have not Switched, checkbox and radios so there were not breaking changes for us :)

Comment (cherniavskii) [reply lib/package.json]: nice ;)

Comment (dmtrKovalenko) [reply lib/package.json]: I have updated dependency, not the peer dependency here.

File: lib/package.json (hunk 1 above)
Comment (cherniavskii) [reply lib/package.json]: What do you think about printing info about utils after package is installed? Like ['jss' does it for opencollective support](https://github.com/cssinjs/jss/blob/09f37523dad9622a5f25c69dffe4a4b2e7bee681/package.json#L111) ?

**Comment by dmtrKovalenko** [comment]
@cherniavskii I have added a prompt, and I think we are ready to release new pickers :) 
If you have not any additional questions, lets go ahead!

**Comment by cherniavskii** [comment]
But wait, CI failed :/

**Comment by dmtrKovalenko** [comment]
I have accidentally remove date-fns from lib dev dependencies, which required for tests :)

**Pull request merged**